staging/src/k8s.io/api/autoscaling/v2beta2/types.go

```diff
@@ -247,6 +247,7 @@ const (
 // ObjectMetricSource indicates how to scale on a metric describing a
 // kubernetes object (for example, hits-per-second on an Ingress object).
 type ObjectMetricSource struct {
+	// describedObject describes a kubernetes object
```

Comment:
This appears to be a reference, not a description. Please get a review from someone in the autoscaling SIG to confirm. Same comment for below change.